### PR TITLE
Add documentation page

### PR DESCRIPTION
**Pull request by ketphan02**

### CODE REVIEW

File: api/api/templates/docs.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
+{% load static %}
+<!DOCTYPE html>
```
Comment (Opeyem1a) [inline api/api/templates/docs.html:2]: I thought this page was going to be on the other repo? There is no explicit plan for this repo to be deployed with any kind of pretty URL, so I'm not sure it makes sense for this to be here.

I also really really dislike putting bootstrap, css, and html into this repo that is supposed to be a microservice purely for algorithm logic

**Review by Opeyem1a — CHANGES_REQUESTED** [review]

File: api/api/templates/docs.html (hunk 1 above)
Comment (ketphan02) [reply api/api/templates/docs.html:2]: I see, should we then close this PR and transfer this into another project?

Comment (Opeyem1a) [reply api/api/templates/docs.html:2]: Yes

Comment (Opeyem1a) [reply api/api/templates/docs.html:2]: Yes

**Comment by ketphan02** [comment]
Not suit this PR, move to different one

**Pull request closed without merging**